Packages.props

```diff
@@ -15,6 +15,8 @@
 		<PackageReference Update="Microsoft.Extensions.DependencyModel" Version="3.1.4" />
 		<PackageReference Update="Microsoft.Extensions.FileSystemGlobbing" Version="7.0.0" />
 
+		<PackageReference Update="Microsoft.Identity.Client" Version="4.49.1" />
```

Comment:
can you confirm these dependencies are compatible with macos arm64?  I assume they likely are but these has been a problem with other dependencies we're trying to fix.

Reply:
We're building x64 arch in ARM64 bits, so there should be no concerns. cc @alanrenmsft